blackjack.py

```diff
@@ -138,15 +143,15 @@ def print_hand_and_value(self):
         for card in self.hand:
             print card
         soft_value = self.get_soft_hand_value()
-        hard_value = self.get_hard_hand_value()
+        hard_value = self.get_hand_value()
         if hard_value == 21.1 or soft_value == 21.1:
             print '\nHand value is Natural 21!'
         elif hard_value == soft_value or hard_value >= 22:
             print '\nHand value is {}'.format(self.get_soft_hand_value())
         else:
             print '\nSoft hand value is {} ' \
                   '\nHard hand value is: {}'.format(self.get_soft_hand_value(),
-                                                    self.get_hard_hand_value())
+                                                    self.get_hand_value())
```

Comment:
You already have the values of soft hand and hand in local variables, you shouldn't call those methods again.

Reply:
Fair, will change.